src/catalog/controller/payment/omise.php

```diff
@@ -75,7 +75,8 @@ public function checkout() {
 							"currency"    => $this->currency->getCode(),
 							"description" => $this->request->post['description'],
 							"return_uri"  => $this->url->link('payment/omise/checkoutcallback&order_id='.$order_id),
-							"card"        => $this->request->post['omise_token']
+							"card"        => $this->request->post['omise_token'],
+							"capture"     => $this->config->get('omise_auto_capture')
```

Comment:
In this part, we do not need to check the available of this config because its initial value is null. It will be not sent to Omise API cause the charge will be captured that is consistent with Omise API.

Check or no check the null value, it provides the same result so I decided to go with this design.

Before this commit, I thought like you and also check the null value like the link you mentioned but after review I think it too duplicate because there are 2 places for the default value. (back-end admin and front-end catalog). That is I think this design is better.